### PR TITLE
Feature/html skeleton

### LINKED ISSUE #5: Create Story Component
Skeleton HTML and CSS component with spots where data will be plugged in later.

### LINKED ISSUE #9: Create Day/Response Component
Skeleton HTML and CSS component with spots where data will be plugged in later.

### LINKED ISSUE #10: Create Choices Component
Skeleton HTML and CSS component with spots where data will be plugged in later.

### PR DESCRIPTION
**Pull request by jsahim**
**Change summary:** 

- Added Frame image with basic styling.
- Added App, Choices, Day, Result and Story components with skeletons.
- Removed extraneous files from Create React App template.

**Functions/methods:** 
N/A

Close #10 Close #9 Close #5

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,11 @@
...
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React Redux App</title>
+    <title>A Cat's Tail</title>
   </head>
```
Comment (qrispi) [inline public/index.html:7]: WOW react loves to talk!

**Comment by Jesusborjas006** [comment]
Great start adding the template and basic styling to page.

File: src/images/frame-backdrop.png
Comment (qrispi) [inline src/images/frame-backdrop.png:1]: I love this image!

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+  padding: 0;
+  box-sizing: border-box;
```
Comment (qrispi) [inline src/index.css:4]: Jesus this is a game changer!

**Comment by BriBourassa** [comment]
Great start everyone! Jeff that is an awesome border

**Review by qrispi — APPROVED** [review]
We are off to a great start! Super excited about this!

**Review by BriBourassa — APPROVED** [review]

File: src/components/App/App.css
Comment (Jesusborjas006) [inline src/components/App/App.css:1]: Love the background!

**Review by Jesusborjas006 — APPROVED** [review]

**Pull request merged**